cfgov/v1/atomic_elements/organisms.py

```diff
@@ -560,3 +560,21 @@ class VideoPlayer(blocks.StructBlock):
     class Meta:
         icon = 'media'
         template = '_includes/organisms/video-player.html'
+
+
+class HTMLBlock(blocks.StructBlock):
+    html_url = blocks.RegexBlock(
+        label='Source URL',
+        default='',
+        required=True,
+        regex=r'^https://.+$',
```

Comment:
we have the files deploying to s3 so we know they will be at http://files.consumerfinance.gov/ now - can you go ahead and add this?